Detections/Corelight/CorelightSMTPEmailSubjectNonAsciiCharacters.yaml

```diff
@@ -0,0 +1,34 @@
+id: 50c61708-9824-46f3-87cf-22490796fae2
+name: Corelight - SMTP Email containing NON Ascii Characters within the Subject
+description: |
+  'Detects where an emails contain non ascii characters within the Subject.'
+severity: Low
+requiredDataConnectors:
+  - connectorId: Corelight
+    dataTypes:
+      - Corelight_CL

```

Comment:
If we are using parser in query datatype should be parser name. So could you please update the datatype name to Corelight.

Reply:
updated